Templates/Tests/Generated/AutoMockable.generated.swift

```diff
@@ -423,6 +423,90 @@ class AnyProtocolMock: AnyProtocol {
     }
 
 
+}
+class AnyProtocolWithOptionalsMock: AnyProtocolWithOptionals {
+
+
+    var a: [(any StubProtocol])?
```

Comment:
a closing parenthesis is added at the wrong place. Similar problem is seen in other cases in this class